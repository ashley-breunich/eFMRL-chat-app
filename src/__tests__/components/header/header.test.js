import React from 'react';
import renderer from 'react-test-renderer';
import Header from 'components/header/header.js';

describe('<Header />', () => {
  it('is alive at application start', () => {
    let app = shallow(<Header />);
    expect(app.find('header').exists()).toBeTruthy();
  });

  it('renders correctly', () => {
    const header = renderer.create(<Header />).toJSON();
    expect(header).toMatchSnapshot();
  });
});
