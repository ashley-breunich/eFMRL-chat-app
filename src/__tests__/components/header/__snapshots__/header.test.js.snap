// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> renders correctly 1`] = `
<header>
  <img
    alt="efmrl logo"
    src="efmrl-chat-final.png"
  />
</header>
`;
